### PR TITLE
Introduce nightly FIPS EE agent parallel tests

### PR DESCRIPTION
**Pull request by uCatu**
### What does this PR do?
Introduce **nightly** FIPS EE agent parallel tests run all agents and environment, currently FIPS EE agents, at nightly triggerd job
- [Success Standard build run blue ocean](https://jenkins.conjur.net/blue/organizations/jenkins/cyberark--conjur/detail/nightly-fips/2/pipeline/)
- [Success EE nightly build run blue ocean](url)
- [Failed EE nightly build run blue ocean](https://jenkins.conjur.net/blue/organizations/jenkins/cyberark--conjur/detail/nightly-fips/6/pipeline/233/)

image
image

#### Change log
- [ ] The CHANGELOG has been updated, or
- [X] This PR does not include user-facing changes and doesn't require a CHANGELOG update

#### Test coverage
- [ ] This PR includes new unit and integration tests to go with the code changes, or
- [X] The changes in this PR do not require tests

#### Documentation
- [ ] Docs (e.g. `README`s) were updated in this PR, and/or there is a follow-on issue to update docs, or
- [X] This PR does not require updating any documentation

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -66,23 +75,93 @@ pipeline {
...
-        }
-        stage('LDAP Authenticator') {
-          steps { sh 'ci/test cucumber_authenticators_ldap' }
-        }
-        stage('OIDC Authenticator') {
-          steps { sh 'ci/test cucumber_authenticators_oidc' }
-        }
+          stage('EE FIPS agent tests') {
+            agent { label 'executor-v2-rhel-ee' }
+            when {
```
Comment (hughsaunders) [inline Jenkinsfile:83]: If you use getTrigger to determine if the build was started by a timer, do you still need parameterised cron and the NIGHTLY parameter?
```suggestion
            when { expression { return getTrigger() == "timer" } }
```

**Review by eladkug — COMMENTED** [review]
There is nice solution for the integration test report. 
Looks like all our reports will contain duplicates reports and sometimes it will be hard to get the input from there. This is an example of unit test:
image

I dont see this for now as a blocker since we can get the input from the pipeline.

File: Jenkinsfile (hunk 1 above)
Comment (uCatu) [reply Jenkinsfile:83]: I would like to have the option to trigger the job manually and get EE suite tests running, to allow developers a quick tool for EE coverge testing, WDYT? 

Timer can be defined for different reasons, even on feature branch - for example running 30 builds by trigger should not invoke EE tests

**Review by uCatu — COMMENTED** [review]
answer

**Review by eladkug — APPROVED** [review]
LGTM from test perspective

**Review by liavyona — APPROVED** [review]
LGTM

**Comment by uCatu** [comment]
@hughsaunders Leaving the branch open - if changes needed I'll revert and update

**Pull request merged**